lib/models/BucketInfo.ts

```diff
@@ -144,6 +144,7 @@ export default class BucketInfo {
         objectLockEnabled?: boolean,
         objectLockConfiguration?: any,
         notificationConfiguration?: any,
+        tags?: { key: string; value: string }[] | null,
```

Comment:
Instead of defaulting to null why not default to an empty array? That is, tags are required but can be empty, rather than optionally null. `if (tags)` will still as well. For compatibility you can do: 
```
if (tags === undefined) { tags = [] }
```

Reply:
Agree and done in https://github.com/scality/Arsenal/pull/1933/commits/08703718b843b40a802adb9aaafdf7840cd05baa#diff-4ee3cd5de98073dadb72b27f034f4979480c04388cc4f4538f4895373527f174R63